src/components/loginComponent.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
+import PropTypes from 'proptypes';
+import Input from './Input';
+import Loader from './Loader';
+import logo from '../assets/img/logo.png';
+
+class loginComponent extends Component {
+    state = {
+        email: '',
+        password: '',
+    }
+    
+    handleChange = (event) => {
+        this.setState({ [event.target.name]: event.target.value });
+    }
+
+    onLogin = (event) => {
+        event.preventDefault();
+        const { login } = this.props;
+        login(this.state)
+    }
+
+    render() {
+        const { loading, success, error, data } = this.props;
+        const { email, password } = this.state;
+
+        if (success) {
+            localStorage.setItem('user', JSON.stringify(data));
+        }
+        return(
+            <div id="body" className="login-body">
+                <div className="form-content">
+                    {success ? <Redirect to="/login.html" /> : null}
```

Comment:
`/login.html` is that the actual path?

Reply:
No. I am yet to work on the component to redirect to.